src/cargo/ops/cargo_run.rs

```diff
@@ -104,6 +104,10 @@ pub fn run(
         process.display_env_vars();
     }
 
+    if let Some(client) = config.jobserver_from_env() {
+        process.inherit_jobserver(client);
```

Comment:
I'm assuming this works fine (based on the tests) when `exec_replace` both does a replace and doesn't.

Reply:
Yes. That also what I understand about `Command::exec`. See <https://github.com/rust-lang/rust/blob/b5c050febf10c9bcc0459d41fe2a1e190ad30b8d/library/std/src/sys/unix/process/process_unix.rs#L249-L270>.